app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  # def index
+  #   @users = User.all
+  # end
+
+  # def show
+  #   @user = User.find_by_id(params[:id])
+  # end
+end
```

Comment:
- [ ] To stay in line with the Microverse General requirements please avoid commenting out pieces of code, this will make your code look unorganised and harder to read. I noticed you made this change after the change request from the previous CR and you probably are planning to implement this code in the next milestone, however, I advise you in this case to remove the code entirely from this branch and add it to a separate branch.